pkg/k8shandler/configmaps.go

```diff
@@ -55,7 +55,7 @@ func CreateOrUpdateConfigMaps(dpl *v1alpha1.Elasticsearch) (err error) {
 		dpl.Namespace,
 		dpl.Labels,
 		kibanaIndexMode,
-		esUnicastHost(dpl.Name),
+		esUnicastHost(dpl.Name, dpl.Namespace),
```

Comment:
You aren't doing anything with the return value of `esUnicastHost`?

Reply:
its being passed along in a function here